frontend/src/learning/components/ItemDisplay/ProgressBar.js

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+
+// Body: use padding to adjust Bar's position 
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 30px;
+`
+// Outer bar container
+const BarContainer = styled.div`
+  background-color: #d8d8d8;
+	position: relative;
```

Comment:
This line was indented with 1 tab, please use 4 spaces instead

Reply:
@Faradawn you should set up ur IDE to do this automatically